tests/kafkatest/tests/core/round_trip_fault_test.py

```diff
@@ -98,9 +98,9 @@ def test_produce_consume_with_latency(self):
         workload1 = self.trogdor.create_task("workload1", self.round_trip_spec)
         time.sleep(2)
         node_specs = {}
```

Comment:
nit: maybe take this variable out as its not used anymore.

Reply:
done